backend/src/test/resources/fhir/observationCorrection.json

```diff
@@ -6,7 +6,7 @@
     "coding": [
       {
         "system": "http://loinc.org",
-        "code": "96741-4"
```

Comment:
For these changes where we are updating the code, is this only for FHIR related events or are we eventually going to change to the new code throughout the codebase? Hope this makes sense 😅

Reply:
Specifically this `observation.code` is only being changed because the value before was incorrect as it was using the `support device loinc` instead of the `device type loinc`. 

Eventually I believe we will move away from using `device.supportedDiseases` and `device.loinc` since those are both values in the new `device.supportedDiseaseTestPerformed`